Assets/Prefabs/PFB_Cursor/CursorController.cs

```diff
@@ -101,7 +101,6 @@ private void OnClick()
             {
                 if (authorities[(int)Actions.Reveal])
                 {
-                    ClientSend.RevealTarget(1);
                 }
```

Comment:
so we just won't do anything in this block now? better to not leave weird empty blocks and just modify the if case completely imo